creditcoin-js/src/extrinsics/register-transfers.ts

```diff
@@ -63,18 +61,20 @@ export const registerRepaymentTransfer = async (
 };
 
 export const verifiedTransfer = async (api: ApiPromise, transferId: TransferId, timeout = 20_000) => {
```

Comment:
the timeout is no longer being used, should it be passed to `listenForVerificationOutcome`?